src/card/basic/card-basic.html

```diff
@@ -0,0 +1,9 @@
+<div ng-class="showTopBorder === 'true' ? 'card-pf card-pf-accented' : 'card-pf'" class="cardContainer">
```

Comment:
Yes you are right. It was only for testing purposes but I can access it via card-pf instead